src/lib/email.js

```diff
@@ -0,0 +1,27 @@
+import sendmail from 'sendmail';
+import StudentStore from '../store/student-store';
+
+const mailer = sendmail({
+    silent: true
+});
+
+export function sendMail(emails){
+    return new Promise((res, rej) => {
+        mailer({
+            from: 'testing@msoe.edu',
+            to: emails.join(','),
+            subject: 'ITEMS DUE SOON',
```

Comment:
Nit: This subject line looks spammy. Maybe `'MSOE Tech Support Items Due Soon'` or just `'Items Due Soon'`?